packages/expo-print/android/src/main/java/expo/modules/print/PrintPDFRenderTask.java

```diff
@@ -83,11 +83,11 @@ private PrintAttributes getPrintAttributes() {
       int width = DEFAULT_MEDIA_WIDTH;
       int height = DEFAULT_MEDIA_HEIGHT;
 
-      if (mOptions.containsKey("width")) {
-        width = (Integer)mOptions.get("width");
+      if (mOptions.containsKey("width") && mOptions.get("width") != null) {
+        width = (int) mOptions.get("width");
```

Comment:
Shouldn't it be `((Number) mOptions.get("width")).intValue()`? Does casting from `Object` to a primitive type `int` work?